packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -65,3 +65,15 @@
 .edit-site-global-styles-color-palette-panel {
 	padding: $grid-unit-20;
 }
+
+.edit-site-global-styles-sidebar__beta {
```

Comment:
This could be some kind of pill component maybe or something.

Reply:
I believe @sarayourfriend started working on a `Tag` component, but the PR (https://github.com/WordPress/gutenberg/pull/32214) was closed prematurely

Reply:
cc @mtias and @jasmussen in case they have more to add on such a potential component

Reply:
This is super temporary, we'll remove it shortly. It doesn't even need to be rendered as a tag, can be just `Styles (Beta)` in text. If we consider using it more for specific blocks and in the long term we can revisit.